sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/RemoteDependencyDataTests.cs

```diff
@@ -4,11 +4,10 @@
 using System;
 using System.Diagnostics;
 using System.Globalization;
-using Azure.Monitor.OpenTelemetry.Exporter.Models;
 using OpenTelemetry.Trace;
 using Xunit;
 
-namespace Azure.Monitor.OpenTelemetry.Exporter
+namespace Azure.Monitor.OpenTelemetry.Exporter.Models.Tests
```

Comment:
Maybe still `Azure.Monitor.OpenTelemetry.Exporter.Tests`? (Judging by the folder name).

Reply:
Sorry - I merged at same time you commented I think. Azure.Monitor.OpenTelemetry.Exporter.Models is the namespace of class this file is testing. Should probably move to a different folder for clarity.

Reply:
No worries, I noticed the race condition after I clicked the approve button 😄

Having a different folder (e.g. `Models`) sounds reasonable 👍